components/Faq.js

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components'
+import FaqSection from './FaqSection'
+
+const FaqContainer = styled.div`
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    gap: 76px;
+    margin-top: 64px;
+
+    ${p => p.theme.mediaQueries.mobile} {
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+    }
+`
+
+const FlexContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+
+    & > div:not(:first-child) {
+        margin-top: 40px;
+
+        ${p => p.theme.mediaQueries.mobile} {
+            margin-top: 24px;
+        }
+    }
+`
+
+export default function Faq({ faqs }) {
+    // Bucket each FAQ into a dictionary where the key is the category
+    var categories = {};
+    faqs.forEach((faq) => {
+        if (categories[faq.category] == undefined) {
+            categories[faq.category] = [];
+        }
+        categories[faq.category].push(faq);
+    });
+
+    return (
+        <FaqContainer>
+            <FlexContainer>
+                {categories['Hackers'] && <FaqSection category={'Hackers'} faqs={categories['Hackers']}/>}
+                {categories['Sponsors'] && <FaqSection category={'Sponsors'} faqs={categories['Sponsors']}/>}
```

Comment:
Are we sure the question categories will be these? On the figma the categories are these 3, but on our production db the FAQ categories are something else (General, Logistics, Team & projects...)

Reply:
oh wait, those FAQs are for other hackathons. I assume we will just be creating new FAQs for `www` with the right categories. That works!